src/rating/Rating.js

```diff
@@ -255,4 +320,7 @@ Rating.propTypes = {
   onFinishRating: PropTypes.func,
   showRating: PropTypes.bool,
   style: View.propTypes.style,
+  readonly: PropTypes.bool,
+  startingValue: PropTypes.number,
+  fractions: fractionsType,
```

Comment:
This is great! The `readonly` and `startingValue` props have been on my to-do list. Glad you got to it, and the implementation looks pretty clean too. 👏